css-three-column-layout/styles.css

```diff
@@ -0,0 +1,56 @@
+.container {
+  background-color: #1996da;
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: 10%;
+  padding-right: 10%;
+  padding-top: 2%;
+}
+
+.row {
+  display: flex;
+  justify-content: space-evenly;
+}
```

Comment:
You should also separate this `justify-content` into it's own class like `justify-evenly` and then give it the justify-content space evenly as it's only propertyl.